src/foundry/client/pixi/extensions/polygon.d.mts

```diff
@@ -123,20 +114,31 @@ declare module "pixi.js" {
   }
 
   namespace Polygon {
+    /**
+     * @privateRemarks Foundry uses this type instead of full `ClipperLib.IntPoint` objects
+     */
     interface ClipperPoint {
       X: number;
       Y: number;
     }
 
     /** @internal Helper type for interface to simplify InexactPartial usage */
     type _IntersectClipperOptions = InexactPartial<{
-      /** The clipper clip type */
-      clipType: number;
-
-      /** A scaling factor passed to Polygon#toClipperPoints to preserve precision */
+      /**
+       * The clipper clip type
+       * @remarks ClipperLib functions require a `ClipperLib.ClipType` value, but the Foundry functions have `??=` guards for this.
+       * @defaultValue `ClipperLib.ClipType.ctIntersection`
+       */
+      clipType: ClipperLib.ClipType | null;
+
+      /**
+       * A scaling factor passed to Polygon#toClipperPoints to preserve precision
+       * @defaultValue `1`
+       */
       scalingFactor: number;
     }>;
 
+    /** @remarks `scalingFactor` can't be null as it only has defaults provided by `{scalingFactor=1}` where it has any */
```

Comment:
Did you not finish the sentence here? I think "where it has any" can be deleted.

Reply:
No, that's a valid if convoluted sentence: "...where it has any [defaults]". I expect this to be irrelevant due to larger changes depending on the answer to the comment above this one.